app/tests/spec/views/permissions.js

```diff
@@ -122,10 +122,10 @@ function (chai, $, sinon, p, View, AuthErrors, Metrics, FxaClient,
       it('renders relier info', function () {
         return initView('sign_up')
           .then(function () {
-            assert(!! view.$('#fxa-permissions-header').text().match(SERVICE_NAME),
-              'service name shows in header');
+            assert(!! view.$('p').text().match(SERVICE_NAME),
```

Comment:
using `p` as a selector is pretty brittle to random passers-by like me who may try to add more to the template some day. Maybe use a class name.